packages/fx-core/src/component/coordinator/index.ts

```diff
@@ -265,6 +269,13 @@ class Coordinator {
         if (meArchitecture) {
           feature = `${feature}:${meArchitecture}`;
         }
+        if (
+          inputs.targetFramework === "net8.0" &&
```

Comment:
What if targetFramework has been upgrade to net9.0 or higher, this logic will be break. Since net6.0 and net7.0 is a history and will be retired, it is more safe to use the inversed condition here.

Reply:
Good idea. I'll correct this judgment.